operations/configuration_management/CFS_Flow_Diagrams.md

```diff
@@ -0,0 +1,32 @@
+# CFS Flow
+
+## Single Session Flow
+
+This section covers the components and actions taken when a user or service creates a session using the CFS sessions endpoint.
+
+![Single Session Flow](../../img/operations/CFS_Single_Session_Flow.png)
+
+1. A user creates a CFS configuration.
+1. A user creates a CFS session, causing a session record to be created.
+1. When a session record is created, the CFS-API also posts an event to a Kafka queue.
+1. The CFS-Operator is always monitoring the Kafka queue, and handles events as they come in.
+1. The CFS-Operator creates a Kubernetes job for the session in response to a session creation event.
+1. CFS-Operator monitors the Kubernetes job (not the pod) for completion.
+1. When the Kubernetes job is complete, CFS-Operator updates the session record in the CFS-API.
+
+## Automated Session Flow
+
+This section covers the components and actions taken when a user or service sets a desired configuration for a component, causing automatic configuration to occur.
+
+![Automated Session Flow](../../img/operations/CFS_Automated_Session_Flow.png)
+
+1. A user creates a CFS configuration.
+1. A user sets the desired configuration for some number of components. This may immediately trigger configuration if the component is enabled in CFS and the desired configuration not yet been applied. If so, skip to step 4.
```

Comment:
```suggestion
1. A user sets the desired configuration for some number of components. This may immediately trigger configuration if the component is enabled in CFS and the desired configuration not yet been applied. If that is the case, then skip to [CFS-Batcher creates sessions](#cfs_batcher_creates_sessions).
```